rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java

```diff
@@ -229,10 +239,12 @@ public String getId() {
 
             private String nodeLocker = null;
 
+            private String eventType = null;
+
             Node(String nodeUrl, String nodeState, String nodeInfo, long timeStamp, String timeStampFormatted,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 17 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)